.dir-locals.el

```diff
@@ -0,0 +1,4 @@
+((c-mode . ((c-file-style . "google")
```

Comment:
What's this file do?

Reply:
It tells emacs to use google style for all files in directories under
this one (e.g. for all LDMX software).  I don't want to set this
globally because I also work on projects with other coding styles.